src/pages/FriendGroup/pages/MyGroups.tsx

```
@@ -1,113 +1,115 @@
+        groupId: quitTarget.id,
+        memberId: myId,
+      });
+
+      setQuitTarget(null);
+      setShowQuitToast(true);
+    } catch (err) {
+      console.error("그룹 탈퇴 처리 중 오류:", err);
+      setQuitTarget(null);
+      setShowErrorToast(true);
+    }
+  }, [myProfile?.id, quitTarget, removeGroupMember]);
```

ESLint

[error] Compilation Skipped: Existing memoization could not be preserved

React Compiler has skipped optimizing this component because the existing manual memoization could not be preserved. The inferred dependencies did not match the manually specified dependencies, which could cause the value to change more or less frequently than expected. The inferred dependency was `myProfile`, but the source dependencies were [myProfile?.id, quitTarget, removeGroupMember]. Inferred less specific property than source.

/home/jailuser/git/src/pages/FriendGroup/pages/MyGroups.tsx:37:41
  35 |   }, []);
  36 |
> 37 |   const handleConfirmQuit = useCallback(async () => {
     |                                         ^^^^^^^^^^^^^
> 38 |     if (!quitTarget) {
     | ^^^^^^^^^^^^^^^^^^^^^^
> 39 |       console.error("탈퇴할 그룹 ID가 유효하지 않습니다.");
     …
     | ^^^^^^^^^^^^^^^^^^^^^^
> 62 |     }
     | ^^^^^^^^^^^^^^^^^^^^^^
> 63 |   }, [myProfile?.id, quitTarget, removeGroupMember]);
     | ^^^^ Could not preserve existing manual memoization
  64 |   return (
  65 |     <>
  66 |       <PageTitle title="나의 그룹 관리" />

(react-hooks/preserve-manual-memoization)